services/php/common/common.sls

```diff
@@ -84,6 +84,26 @@ makina-php-composer:
 
 {{ macros.toggle_ext('xdebug', phpSettings.xdebug_install and phpSettings.xdebug_enabled) }}
 
+{% if phpSettings.opcache_install %}
+makina-php-opcache:
+  file.managed:
+    - user: root
+    - makedirs: true
+    - group: root
+    - mode: 664
+    - name: {{ phpSettings.confdir }}/opcache.ini
+    - source: salt://makina-states/files{{ phpSettings.confdir }}/opcache.ini
+    - template: 'jinja'
+    - defaults:
+        enable: {{ phpSettings.opcache_enabled }}
```

Comment:
now that we have modules registries, we must not use variables in templates, to clarify code.
eg directly in  files/etc/php5/mods-available/opcache.ini:
on the top {{{
{% set phpsettings = salt['mc_php.settings']() %}
}}}
and then use phpSettings.opcache_enabled

This avoid a lot of jinja templating code and avoid rendering bugs.

We should also now not use directly file.managed, but a special macro which simplifies the process like in:
https://github.com/makinacorpus/makina-states/blob/master/services/virt/lxc/configuration.sls#L41